spec/requests/targets/index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'GET /targets', type: :request do
+  let!(:user)  { create(:user) }
+  let!(:user_targets) { create_list(:target, 3, user: user) }
+
+  before { get targets_path, headers: auth_user_headers, as: :json }
+
+  context 'when the user is authenticated and has targets' do
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'returns a list with 3 targets' do
+      expect(json[:targets].count).to eq 3
+    end
+
+    it 'returns the correct information' do
+      expect(json[:targets]).to include_json(
```

Comment:
I think it should be `match_array` instead of `include_json`

Reply:
If you use ```match_array``` rspec returns an error similar to this:
```
       expected collection contained:  [{:categories=>[{:color=>"#2d6126"}]}]
       actual collection contained:    [{"categories"=>[{"color"=>"#2d6126"}]}]
```
Is there a way to solve this error with ```match_array```?